lib/service/storage/elasticsearch.ts

```diff
@@ -971,7 +1083,16 @@ class ElasticSearch extends Service {
    *
    * @returns {Promise}
    */
-  async delete(index, collection, id, { refresh } = {}) {
+  async delete(
+    index: string,
+    collection: string,
+    id: string,
+    {
+      refresh,
+    }: {
+      refresh?: boolean | "wait_for";
+    } = {},
```

Comment:
```suggestion
    {
      refresh,
    }: Pick<KRequestParams, "refresh"> = {},
```

With new `KRequestParams` type we can compose type like it, to DRY